dev/generate_bom.py

```diff
@@ -180,8 +184,10 @@ def generate_changelog(self):
 
       # Generate the changelog for the component.
       print 'Generating changelog for {comp}...'.format(comp=comp)
-      result = run_quick('cd {comp}; clog -f {hash} --setversion {version}; cd ..'
-                         .format(comp=comp, hash=hash, version=version))
+      component_url = ('https://github.com/{owner}/{comp}'
```

Comment:
I dont understand why you are doing this as opposed to using the local source.
The net effect seems to be that you are generating the changelog for something different from what you generated. Either because you put a different name here, or have a race condition in which the source updated on github different from the source you built.